File: engine/engine-model/src/main/scala/net/server/channel/packet/guild/ExpelFromGuildPacket.scala
package net.server.channel.packet.guild

class ExpelFromGuildPacket(private var _type: Byte, private var _playerId: Int, private var _name: String) extends BaseGuildOperationPacket(_type = _type) {
  def playerId: Int = _playerId

  def name: String = _name
}

File: engine/engine-model/src/main/java/net/server/channel/packet/guild/ExpelFromGuildPacket.java
package net.server.channel.packet.guild;

public class ExpelFromGuildPacket extends BaseGuildOperationPacket {
   private final Integer playerId;

   private final String name;

   public ExpelFromGuildPacket(Byte theType, Integer playerId, String name) {
      super(theType);
      this.playerId = playerId;
      this.name = name;
   }

   public Integer playerId() {
      return playerId;
   }

   public String name() {
      return name;
   }
}
